### ISSUE TITLE
GraphQL query cms page content html is emtpy - shortcodes are not resolved 

### ISSUE THREAD

**Issue by 0-Sony** [magento/magento2#26114]
### Preconditions (*)

1. From fresh install: Magento 2.3.2
2. From a Magento 2 demo website using 2.3.2 as well: http://pagebuilder.m2.magezon.com/

### Steps to reproduce (*)

1. use ChromeiQL, set a fresh Magento 2.3.2 endpoint follow by '/graphql'
2. enter this query in ChromeiQL :
 ```
{
  cmsPage(id:2){
    content
  }
}
```
3. use ChromeiQL, set this endpoint: http://pagebuilder.m2.magezon.com/graphql
### Expected result (*)

1. First case (fresh install) : Even though the content field is empty, I expected to be able to get the HTML content with the graphQL query. It seems that we can not get the widgets with graphQL.
2. In the second case (with the http://pagebuilder.m2.magezon.com/) website, the content is filled well. However, shortcodes are not resolved. 

### Actual result (*)

1. the data displays in both case 
```
{
  "data": {
    "cmsPage": {
      "content": ""
    }
  }
}
```

**Comment by engcom-Bravo** [comment]
Reproducible on Magento 2.4 with Sample Data
The GraphQL query of the CMS page with id **2** returns: 
```graphql
{
  "data": {
    "cmsPage": {
      "content": ""
    }
  }
}
```
But the same query for http://pagebuilder.m2.magezon.com/graphql returns: 
```graphql
SyntaxError: Unexpected token < in JSON at position 6
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Bravo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29941` were created

**Issue Available**: @engcom-Bravo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by sanjay-wagento** [comment]
#MM20IN

**Comment by danielrenaud** [comment]
@0-Sony Can you add steps to reproduce from a fresh magento install (preferably a newer version)?
For the case of the sample data homepage. The page's content is actually empty, and widget are added to it via layout updates. So we would expect the result:
```{
  "data": {
    "cmsPage": {
      "content": ""
    }
  }
}```
what querying that page.

**Comment by pmarjan** [comment]
@0-Sony can you please add steps to reproduce the [mgz_pagebuilder] case?

It seems to me, that having a pagebuilder of a certain kind, yes it is adding custom shortcodes, but, I get to see the correct content in the graphql response.

Eg. if content is added in backend like
`<p>&lt;p&gt;[mgz_pagebuilder]content[/mgz_pagebuilder]&lt;/p&gt;</p>`

exactly the same content is what I see in the response, which is what is expected I believe.
 
I tried this on Magento 2.4.

**Comment by mauragcyrus** [comment]
Team was unable to reproduce on 2.4 develop. Closing this ticket.

**Comment by magento-engcom-team** [comment]
Hi @0-Sony. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-29941` was closed as `non-reproducible` in `2.4-develop`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean Magento instance or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.
